security-insights.yml

```diff
@@ -0,0 +1,45 @@
+header:
+  schema-version: 2.0.0
+  last-updated: '2025-04-01'
+  last-reviewed: '2025-04-01'
+  url: https://github.com/open-telemetry
+
+project:
+  name: OpenTelemetry
+  administrators:
+    - name: Alolita Sharma
+      affiliation: Apple
+    - name: Austin Parker
+      affiliation: Honeycomb
+    - name: Armin Ruech
+      affiliation: Dynatrace
+    - name: Bogdan Drutu
+      affiliation: Snowflake
+    - name: Carlos Alberto
+      affiliation: Lightstep
+    - name: Daniel Gomez Blanco
+      affiliation: Skyscanner
```

Comment:
@danielgblanco should we change this? 

Congrats on the new role BTW!

Reply:
```suggestion
      affiliation: New Relic
```